src/modules/authentication/components/Registration/Registration.tsx

```diff
@@ -16,7 +16,7 @@ export default function Registration() {
 
 	const [isRePasswordVisible, setIsRePasswordVisible] = useState<boolean>(false);  
```

Comment:
We can pull the latest changes from the main branch and use the password input component. We can also use the useToggle hook inside it.

Reply:
done